test/e2e/dra/dra.go

```diff
@@ -891,7 +895,7 @@ var _ = framework.SIGDescribe("node")("DRA", feature.DynamicResourceAllocation,
 		b2 := newBuilder(f, driver2)
 		b2.classParameters = driver2Params
 
-		f.It("selects the first subrequest that can be satisfied", feature.DRAPrioritizedList, func(ctx context.Context) {
+		f.It("selects the first subrequest that can be satisfied", func(ctx context.Context) {
```

Comment:
So these cases don't need the feature?

Reply:
From the description:

- old: [sig-node] DRA [Feature:DynamicResourceAllocation] [FeatureGate:DynamicResourceAllocation] [Beta] [Feature:OffByDefault] with prioritized list selects the first subrequest that can be satisfied [Feature:DRAPrioritizedList]
- new: [sig-node] [DRA] [Feature:DynamicResourceAllocation] [FeatureGate:DynamicResourceAllocation] [Beta] [Feature:OffByDefault] [FeatureGate:DRAPrioritizedList] [Alpha] [Feature:OffByDefault] selects the first subrequest that can be satisfied

Reply:
`[Feature:OffByDefault]` appears twice, once for the DynamicResourceAllocation feature, once for DRAPrioritizedList. That's okay and the first one will go away when DRA graduates to GA.